src/i18n/es.js

```diff
@@ -141,15 +141,15 @@ export const messages_es = {
   'editor.container_padding.label': `Relleno Contenedor`,
   'editor.content_alignment.label': `Alineamiento del contenido`,
   'editor.content_background_color.label': `Color de fondo de contenido`,
-  'editor.content_width.label': `Ancho Contenido`,
+  'editor.content_width.label': `Ancho contenido`,
```

Comment:
Según [lo hablado](https://makingsense.slack.com/archives/CLBEN4JAV/p1678994237063369?thread_ts=1678992624.890959&cid=CLBEN4JAV) con @GGossweiler:

```suggestion
  'editor.content_width.label': `Ancho del contenido`,
```